Interfaces/scripts/dirac-dms-lfn-accessURL.py

```diff
@@ -11,39 +11,45 @@
 import DIRAC
 from DIRAC.Core.Base import Script
 
-Script.setUsageMessage( '\n'.join( [ __doc__.split( '\n' )[1],
-                                     'Usage:',
-                                     '  %s [option|cfgfile] ... LFN SE' % Script.scriptName,
-                                     'Arguments:',
-                                     '  LFN:      Logical File Name or file containing LFNs',
-                                     '  SE:       Valid DIRAC SE' ] ) )
-Script.parseCommandLine( ignoreErrors = True )
+Script.setUsageMessage('\n'.join([ __doc__.split('\n')[1],
+                                   'Usage:',
+                                   '  %s [option|cfgfile] ... LFN SE [PROTO]' % Script.scriptName,
+                                   'Arguments:',
+                                   '  LFN:      Logical File Name or file containing LFNs',
+                                   '  SE:       Valid DIRAC SE',
+                                   '  PROTO:    Optional protocol for accessURL']))
+Script.parseCommandLine(ignoreErrors = True)
```

Comment:
```suggestion
Script.parseCommandLine(ignoreErrors=True)
```
Fix for the whitespace

Reply:
Hmm, I clearly missed those, not sure why my pylint didn't pick them up...

Reply:
This is not coming from a `pylint` but a `pycodestyle` check